app/src/main/java/com/duckduckgo/app/browser/WebViewVersionProvider.kt

```diff
@@ -25,6 +25,7 @@ import javax.inject.Inject
 @ContributesBinding(AppScope::class)
 class DefaultWebViewVersionProvider @Inject constructor(
     private val webViewVersionSource: WebViewVersionSource,
+    private val webViewPackageSource: WebViewPackageSource,
```

Comment:
Ditto in another comment. I think the package source can be an implementation detail of this class. No need to create a different class for just one single use?